include/ck_tile/core/arch/amd_buffer_addressing.hpp

```diff
@@ -76,6 +111,11 @@ struct buffer_load<16, pre_nop>
     {
         static_assert(sizeof(T) == 16);
         using mbuf_t = typename impl::buffer_load_trait<16, T>::payload_t;
+#if HAS_RAW_BUFFER_BUILTINS
+        index_t s_offset                 = i_offset;
```

Comment:
It'd be great if we made `i_offset` into a compile-time constant `number<i_offset>`.

Luckily, it seems that `s_offset` and `i_offset` are both always `0` so this works.